hyperion-build-config/src/main/res/layout/hbc_activity_build_config_list.xml

```diff
@@ -8,25 +8,25 @@
     android:focusableInTouchMode="true"
     tools:context=".list.BuildConfigListActivity">
 
-    <android.support.design.widget.AppBarLayout
+    <com.google.android.material.appbar.AppBarLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content">
 
-        <android.support.v7.widget.Toolbar
+        <androidx.appcompat.widget.Toolbar
             android:id="@+id/tmb_toolbar"
             android:layout_width="match_parent"
             android:layout_height="?attr/actionBarSize"
             android:theme="@style/hbc_ActionBar"
             app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
             app:title="@string/hbc_plugin_name" />
 
-    </android.support.design.widget.AppBarLayout>
+    </com.google.android.material.appbar.AppBarLayout>
 
-    <android.support.v7.widget.RecyclerView
+    <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/hbc_recycler"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         app:layout_behavior="@string/appbar_scrolling_view_behavior"
```

Comment:
Is this the correct behavior reference after the androidx change? I know our app had a lot of bad behavior references after upgrading that bit us.

Reply:
Looks like ` @string/appbar_scrolling_view_behavior` is referencing the new `com.google.android.material.appbar.AppBarLayout.ScrollingViewBehavior`